pytest.ini

```diff
@@ -8,3 +8,4 @@ filterwarnings=
 addopts =
 	--cov=twine --cov-context=test --cov-report=
 	--disable-socket
+	-r aR
```

Comment:
Is there a long version of these options (for clarity)?

Reply:
> Is there a long version of these options (for clarity)?

I wish: https://docs.pytest.org/en/stable/usage.html#detailed-summary-report